wifiphisher/extensions/lure10.py

```diff
@@ -0,0 +1,104 @@
+"""
+Extension that implements the Lure10 attack.
+
+Exploits the Wi-Fi Sense feature and will result
+to automatic association by fooling the Windows
+Location Service
+"""
+
+import struct
+import wifiphisher.common.constants as constants
+import scapy.layers.dot11 as dot11
+
+
+class Lure10(object):
+    """
+    Sends a number of beacons to fool Windows Location Service.
+    """
+
+    def __init__(self, shared_data):
+        """
+        Setup the class with all the given arguments.
+
+        :param self: A Lure10 object.
+        :param data: Shared data from main engine
+        :type self: Deauthentication
+        :type data: dictionary
+        :return: None
+        :rtype: None
+        """
+
+        self.first = True
+        self.first_output = True
+        self.data = shared_data
+        self.beacons_num = 0
+
+    def get_packet(self, pkt):
```

Comment:
It can't be removed. The API provides specific data, it's up to the module what to use or not.